tests/src/test/java/alluxio/client/AbstractFileOutStreamIntegrationTest.java

```diff
@@ -62,27 +57,76 @@ public void before() throws Exception {
   }
 
   /**
+   * Helper to write an Alluxio file with stream of bytes of increasing byte value.
+   *
+   * @param filePath path of the tmp file
+   * @param fileLen length of the file
+   * @param op options to create file
+   * @throws Exception if error occurs
+   */
+  protected void writeIncreasingBytesToFile(AlluxioURI filePath, int fileLen, CreateFileOptions op)
+      throws Exception {
+    FileOutStream os = mFileSystem.createFile(filePath, op);
+    for (int k = 0; k < fileLen; k++) {
+      os.write((byte) k);
+    }
+    os.close();
+  }
+
+  /**
+   * Helper to write an Alluxio file with increasing byte array.
+   *
+   * @param filePath path of the tmp file
+   * @param fileLen length of the file
+   * @param op options to create file
+   * @throws Exception if error occurs
+   */
+  protected void writeIncreasingByteArrayToFile(AlluxioURI filePath, int fileLen,
+      CreateFileOptions op) throws Exception {
+    FileOutStream os = mFileSystem.createFile(filePath, op);
+    os.write(BufferUtils.getIncreasingByteArray(fileLen));
+    os.close();
+  }
+
+  /**
+   * Helper to write an Alluxio file with two increasing byte arrays separately.
```

Comment:
Is this writing just one byte array, but using two separate write invocations? I think this could use more elaboration to make it clear what exactly this method is supposed to do.